site/content/en/docs/tasks/run/jobsets.md

```diff
@@ -128,60 +128,8 @@ spec:
 
 ### MultiKueue Environment
 
-```yaml
-# jobset-sample.yaml
-apiVersion: jobset.x-k8s.io/v1alpha2
-kind: JobSet
-metadata:
-  generateName: sleep-job-
-  labels:
-    kueue.x-k8s.io/queue-name: user-queue
-spec:
-  network:
-    enableDNSHostnames: false
-    subdomain: some-subdomain
-  replicatedJobs:
-    - name: workers
-      replicas: 2
-      template:
-        spec:
-          parallelism: 4
-          completions: 4
-          backoffLimit: 0
-          template:
-            spec:
-              containers:
-                - name: sleep
-                  image: busybox
-                  resources:
-                    requests:
-                      cpu: 1
-                      memory: "200Mi"
-                  command:
-                    - sleep
-                  args:
-                    - 100s
-    - name: driver
-      template:
-        spec:
-          parallelism: 1
-          completions: 1
-          backoffLimit: 0
-          template:
-            spec:
-              containers:
-                - name: sleep
-                  image: busybox
-                  resources:
-                    requests:
-                      cpu: 2
-                      memory: "200Mi"
-                  command:
-                    - sleep
-                  args:
-                    - 100s
-  managedBy: kueue.x-k8s.io/multikueue
-```
+The same `jobset-sample.yaml` file from [single cluster environment](#single-cluster-environment) can be used in a [MultiKueue environment](#multikueue-environment).
```

Comment:
nit:  please remove the sections `### Single Cluster Environment` and `### MultiKueue Environment`, and wrap the comment here in a `Note`, like [here](https://github.com/kubernetes-sigs/kueue/blob/85f8c67843fb81a074abe754bf84eda8dcca449e/site/content/en/docs/tasks/manage/setup_multikueue.md?plain=1#L19-L26).

Reply:
lgtm, thanks